README.md

```diff
@@ -9,33 +8,43 @@ following:
  *  Container's logs (STDOUT, STDERR)
  *  Docker stats
 
-Quick Start
------------
 
-```
+## Usage
+
+Build the image with:
+```sh
 docker build -t mu-swarm-logger-service .
+```
 
+Run the container with:
+```sh
 docker run -it --rm \
     --link database:some_container \
     -v /var/run/docker.sock:/var/run/docker.sock \
     mu-swarm-logger-service
 ```
 
-All the containers that have a label `LOG` will be logged to the concept in
-value of this label. For instance:
 
-```
+* All the containers that have a label `LOG` will be logged to the concept in value of this label. For instance for the nginx container:
+
+```sh
```

Comment:
Don't use sh, it's useless (= can be confusing). The format "bash" adds colors but doesn't make it more readable at all so... don't put any format at all.